homework2/sass/common.scss

```diff
@@ -0,0 +1,54 @@
+
+
+$color-secondary-text: #7F7F7F;
+$color-footer-back: #F2F2F2;
+$color-link: #22BF95;
+$color-link-hover: #59D865;
+$color-primary: #FFF;
+$color-primary-text: #191919;
+$color-block-divider: #B2B2B2;
+$color-mark: #E5E5E5;
+
+
+@mixin fontProp($fs,$lh,$c) {
+  font-size: $fs;
+  line-height: $lh;
+  color: $c;
+}
+
+.isHidden{
+  display: none;
+}
+
+@mixin tpl-button($border-color: $color-link, $font-color: $color-link, $bgc: $color-primary, $btn-width: 55px,
```

Comment:
what does `tp1-button` means? also bad idea to set link related variables as default button property value

Reply:
"tpl-button" mean "typical-button".
Changed mixin to class.